code/game/objects/structures/flora.dm

```diff
@@ -411,7 +411,6 @@
 //Jungle rocks
 
 /obj/structure/flora/rock/jungle
-	icon_state = "pile of rocks"
```

Comment:
this should be name =

Reply:
I thought the same thing, but I removed it completely since the rock piles above didn't have names either.
Added as `name` anyways since I doubt there's any harm in it.